ocean_lib/web3_internal/contract_base.py

```diff
@@ -83,7 +84,7 @@ def function_names(self) -> List[str]:
         return list(self.contract.functions)
 
     @staticmethod
-    def to_checksum_address(address: str):
+    def to_checksum_address(address: str) -> str:
```

Comment:
Return value should be `ChecksumAddress`

Reply:
https://web3py.readthedocs.io/en/stable/web3.main.html?highlight=tochecksumaddress#web3.Web3.toChecksumAddress it seems to be a str

Reply:
https://github.com/ethereum/web3.py/blob/master/web3/main.py#L221

Reply:
I think this can be resolved now :)